src/mainloop.c

```diff
@@ -9,13 +9,13 @@
 #include <sys/socket.h>
 #include <unistd.h>
 
+#include "flag.h"
```

Comment:
Let's move the code in file `flag.h` into file `src/mainloop.c`.

Reply:
OK, please wait a second.

Reply:
When I wrote the commit message like `Move the file from src/flag.h to src/mainloop.c`, commit-msg.hook always print warning message `Possible misspelled word(s): src`, is there any suggest for modifying the commit message? Thank you!

![image](https://user-images.githubusercontent.com/42119758/174327473-5352becd-8775-4f4a-a574-dae2b7455046.png)

Reply:
> When I wrote the commit message like `Move the file from src/flag.h to src/mainloop.c`, commit-msg.hook always print warning message `Possible misspelled word(s): src`, is there any suggest for modifying the commit message? 

Always write down informative messages. You can say,
> Unify command line option handling in main loop

It is not vital to mention file names since git itself always tracks the metadata.